src/main/java/org/terasology/launcher/gui/javafx/SettingsController.java

```diff
@@ -116,9 +112,9 @@
     @FXML
     private Button cancelSettingsButton;
     @FXML
-    private ComboBox<JobItem> jobBox;
+    private ComboBox<ApplicationController.PackageItem> jobBox;
     @FXML
-    private ComboBox<VersionItem> buildVersionBox;
+    private ComboBox<String> buildVersionBox;
```

Comment:
Using `Strings` as identifiers can bring some problems (including checks for `null` and empty string, dealing with case sensitivity when comparing, etc). In the long run, we should have a custom class for this purpose as before.